policy/modules/contrib/rhcd.te

```diff
@@ -0,0 +1,36 @@
+policy_module(rhcd, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type rhcd_t;
+type rhcd_exec_t;
+init_daemon_domain(rhcd_t, rhcd_exec_t)
+
+type rhcd_unit_file_t;
+systemd_unit_file(rhcd_unit_file_t)
+
+type rhcd_var_run_t;
+files_pid_file(rhcd_var_run_t)
+
+permissive rhcd_t;
+
+########################################
+#
+# rhcd local policy
+#
+allow rhcd_t self:fifo_file rw_fifo_file_perms;
+allow rhcd_t self:unix_stream_socket create_stream_socket_perms;
+
+manage_dirs_pattern(rhcd_t, rhcd_var_run_t, rhcd_var_run_t)
```

Comment:
I believe files_pid_filetrans() is needed unless the directory is created by some other process.

Reply:
Fixed, thank you